source/js/components/join/language-select.jsx

```diff
@@ -36,13 +36,19 @@ export default class LanguageSelect extends React.Component {
     let classes = classNames(`form-control`, this.props.className);
 
     return (
-      <select
-        value={this.props.selectedLang}
-        onChange={evt => this.handleChange(evt)}
-        className={classes}
+      <div
+        id={`userLanguage-${this.props.formPosition}`}
```

Comment:
What's this prop about and where does it get passed in?

Reply:
It gets passed in [here](https://github.com/mozilla/foundation.mozilla.org/pull/4018/files#diff-bcd043d2cebc25df2ecfd6167302e3c8R398); I added that prop to the `id` since multiple `JoinUs` components can appear on a page and don't want any `id` errors 😬 .

<img width="1440" alt="Screen Shot 2019-12-05 at 3 55 55 PM" src="https://user-images.githubusercontent.com/36484608/70273351-e5130f00-1777-11ea-9edb-8dc2529001b3.png">